scripts/Muon/GUI/Common/muon_context.py

```diff
@@ -4,17 +4,136 @@
 #     NScD Oak Ridge National Laboratory, European Spallation Source
 #     & Institut Laue - Langevin
 # SPDX - License - Identifier: GPL - 3.0 +
-from Muon.Gui.Common.muon_data_context import MuonDataContext
+from Muon.GUI.Common.muon_data_context import MuonDataContext
+from Muon.GUI.Common.muon_gui_context import MuonGuiContext
+from Muon.GUI.Common.muon_group_pair_context import MuonGroupPairContext
+from Muon.GUI.Common.calculate_pair_and_group import calculate_group_data, calculate_pair_data, estimate_group_asymmetry_data
+from Muon.GUI.Common.utilities.run_string_utils import run_list_to_string
+
+from Muon.GUI.Common.ADSHandler.workspace_naming import (get_raw_data_workspace_name, get_group_data_workspace_name,
+                                                         get_pair_data_workspace_name, get_base_data_directory,
+                                                         get_raw_data_directory, get_group_data_directory,
+                                                         get_pair_data_directory, get_group_asymmetry_name)
 
 
 class MuonContext(object):
-    def __init__(self):
-        self._data_context = MuonDataContext()
+    base_directory = "Muon Data"
```

Comment:
lets make this an input, then we can easily have separate frequency and time. May also be worth passing a second string to use to make the other names unique. Or we could do it later

Reply:
Agreed on the first point. I think passing a second string to make the other names unique is also a good idea but wary of making the current PR larger than it already is. This change would also be constrained to just the renaming functions so shouldn't be too difficult to do in the future.

Reply:
lets make an issue and do it later then